src/lib/builders/hover-card/create.ts

```diff
@@ -145,34 +164,33 @@ export function createHoverCard(props: CreateHoverCardProps = {}) {
 				}
 			};
 
-			const portalReturn = usePortal(node);
-
 			let unsubPopper = noop;
-			const unsubOpen = open.subscribe(($open) => {
-				if ($open) {
-					tick().then(() => {
-						const triggerEl = document.getElementById(ids.trigger);
-						if (!triggerEl || node.hidden) return;
-						const $positioning = get(positioning);
-						const $closeOnOutsideClick = get(closeOnOutsideClick);
 
+			const unsubDerived = effect(
+				[isVisible, positioning, closeOnOutsideClick, portal],
+				([$isVisible, $positioning, $closeOnOutsideClick, $portal]) => {
+					unsubPopper();
+					if (!$isVisible) return;
+					const $activeTrigger = get(activeTrigger);
```

Comment:
Why not add activeTrigger as a dependency?

Reply:
@TGlide I could, but I want to say when I tried that it was being a bit weird due to `isVisible` also being a dependency of `activeTrigger`. Will investigate further.